rke/templates/calico_v3.22-rancher2.go

```diff
@@ -0,0 +1,4643 @@
+package templates
+
+/*
+CalicoTemplateV3_22_5 is based on upstream calico v3.22.5
+Upstream Changelog:
+- Add some new properties in crds
+- Add an initcontainer for BPF data plane in calico-node daemonset
+Rancher Changelog:
```

Comment:
Since upstream calico doesn't have this change for v3.22.5, let's call it out in rancher changelog. (Same for v2.6 PR) 
```
Added preserveUnknownFields: false for felixconfigurations and ipamblocks
```